src/Security/Authentication/OAuth/src/OAuthOptions.cs

```diff
@@ -14,7 +14,7 @@ public class OAuthOptions : RemoteAuthenticationOptions
     /// <summary>
     /// Initializes a new instance of <see cref="OAuthOptions"/>.
     /// </summary>
-    public OAuthOptions()
+    public OAuthOptions() : base()
```

Comment:
https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/using-constructors

> In a derived class, if a base-class constructor isn't called explicitly by using the base keyword, the parameterless constructor, if there's one, is called implicitly. The following constructor declarations are effectively the same:

```C#

public Manager(int initialData)
{
    //Add further instructions here.
}

public Manager(int initialData)
    : base()
{
    //Add further instructions here.
}
```

The change you've made has no effect. I suggest closing this PR and we troubleshoot the original problem in the linked issue.